arches/app/datatypes/datatypes.py

```diff
@@ -1041,7 +1042,26 @@ def update(self, tile, data, nodeid=None, action=None):
         updated_data = tile.data[nodeid]
         return updated_data
 
+    def find_num(self, current_item):
+            if len(current_item) and type(current_item[0]) == float:
```

Comment:
If you isinstance() instead of check equality or identity, you'll be flexible enough in case somebody uses a numpy float.

Asserting against an exact type is usually a problem.